chains/ideal-network/node/src/chain_spec.rs

```diff
@@ -148,7 +148,20 @@ pub fn local_testnet_config() -> ChainSpec {
 				.unwrap()
 				.into(),
 		],
-		vec![],
+		vec![
+			get_account_id_from_seed::<sr25519::Public>("Alice"),
```

Comment:
I think the expected deriviation path is `//Alice` and so on (e.g. https://github.com/paritytech/polkadot-sdk/blob/4e122d496f68a820e0e13f63435b6d74fb53be0d/substrate/primitives/core/src/sr25519.rs#L610)